# Generated from: /home/atty/Prog/__IGN__/2015_LI3DS/__ROS__/Ellipse_N/ROS/overlay_ws/src/sbg_ros_driver/sbgECom/src/binaryLogs/sbgEComBinaryLogGps.h
# SbgLogGpsVel
uint32 timeStamp
uint32 status
uint32 timeOfWeek
float32[3] velocity
float32[3] velocityAcc
float32 course
float32 courseAcc